api/v1beta1/hyperconverged_types.go

```diff
@@ -450,6 +450,17 @@ type HyperConvergedFeatureGates struct {
 	// +kubebuilder:default=false
 	// +default=false
 	AutoResourceLimits *bool `json:"autoResourceLimits,omitempty"`
+
+	// Enable KubeVirt emulator thread to assign two extra CPUs if needed to complete even parity.
+	// If IsolateEmulatorThread is set on the VMI, kubevirt assigns extra CPUs to the emulator thread.
+	// By default, KubeVirt only assigns one emulator thread CPU to the virt-launcher pod.
+	// When this feature gate is enabled, and the VMI enables IsolateEmulatorThread, KubeVirt will
+	// complete to assign even parity of CPUs.
+	// Note: this feature is in Developer Preview.
+	// +optional
+	// +kubebuilder:default=false
+	// +default=false
+	AlignCPUs *bool `json:"AlignCPUs,omitempty"`
```

Comment:
Please start json names with lower-cases; i.e.
```suggestion
	AlignCPUs *bool `json:"alignCPUs,omitempty"`
```

Reply:
DONE